### ISSUE TITLE
Unable to stream cloudwatch logs

### ISSUE THREAD

**Issue by mujahedalaqqad** [awslabs/aws-codebuild-jenkins-plugin#114]
Suddenly the plugin is no longer able to stream logs from cloudwatch 

```
awsCodeBuild credentialsType: 'keys', awsAccessKey: "${awsAccessKey}", awsSecretKey: "${awsSecretKey}",
        projectName: project, region: 'us-east-1', exceptionFailureMode: 'DISABLED',
        cwlStreamingDisabled: 'False', sourceControlType: 'project', sourceTypeOverride: 'BITBUCKET', sourceVersion: gitReference, downloadArtifacts: 'false'
```

I tried to invoke the get-event cloudwatch API manually using the aws cli (aws logs get-log-events) just to make sure that the permissions are setup correctly, and it worked but the plugin itself can't stream logs.

also another thing I noticed is that the plugin is not able to stop an in-progress build in codebuild when I abort the jenkins build

<img width="1412" alt="jenkins" src="https://user-images.githubusercontent.com/20612263/135977913-c1ebff58-4f42-467b-87c0-b0bf376586a0.png">

**Comment by denis-matchenko** [comment]
The same issue:
Jenkins: 2.314
AWS CodeBuild: 0.53

**Comment by denis-matchenko** [comment]
Downgrade `aws-java-sdk` to version `1.12.68` fixed the problem
https://plugins.jenkins.io/aws-java-sdk/

**Comment by leoherran-aws** [comment]
Thanks for the heads up. I've reverted the change introduced in 0.53 and will release 0.54 today.

**Comment by mujahedalaqqad** [comment]
> Downgrade `aws-java-sdk` plugin to version `1.12.68` fixed the problem
> https://plugins.jenkins.io/aws-java-sdk/

Yes, it works only when downgrading, thank you

**Comment by denis-matchenko** [comment]
> Thanks for the heads up. I've reverted the change introduced in 0.53 and will release 0.54 today.

Still not working, versions:
`Jenkins: 2.314`
`AWS CodeBuild Plugin (aws-codebuild): 0.54`
`Amazon Web Services SDK :: All (aws-java-sdk): 1.12.70`
`Amazon Web Services SDK :: CodeBuild (aws-java-sdk-codebuild): 1.12.70`

**Comment by denis-matchenko** [comment]
The new version `1.12.80` of the `aws-java-sdk` plugin fixed the problem.

**Comment by leoherran-aws** [comment]
Thanks for the update--I've also confirmed with some other users that upgrading the `aws-java-sdk` plugin to `1.12.80` solves the issue. I will update the plugin dependencies and release a new version today.

**Comment by pandrel** [comment]
@leoherran-aws @denis-matchenko Thank you so much .. I was running into same issue and troubleshooting from almost 4 days. Upgrading aws-java-sdk to 1.12.80 resolved my issue. - What a relief :)

**Comment by leoherran-aws** [comment]
I just released version `0.55` which has a direct dependency on version `1.12.80` of the `aws-java-sdk` plugin. Should be available in a few hours.
